src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+public class FileWriterImpl implements FileWriter {
+    private static final String FILE_REPORT_PATH = "src/main/resources/fileReport.csv";

```

Comment:
Do not hardcode the path, it's better to pass it as argument in the writeToFile method